rcsb/workflow/targets/ProteinTargetSequenceExecutionWorkflow.py

```diff
@@ -31,16 +31,30 @@
 
 
 class ProteinTargetSequenceExecutionWorkflow(object):
-    def __init__(self):
-        self.__mockTopPath = None
-        configPath = os.path.join(HERE, "exdb-config-example.yml")
-        configName = "site_info_remote_configuration"
-        self.__cfgOb = ConfigUtil(configPath=configPath, defaultSectionName=configName, mockTopPath=self.__mockTopPath)
-        self.__cachePath = os.path.join(HERE, "CACHE")
+    def __init__(self, **kwargs):
+        """Workflow wrapper  --  Workflow to rebuild and stash "buildable" cache resources.
+
+        Args:
+            configPath (str, optional): path to configuration file (default: exdb-config-example.yml)
+            configName (str, optional): configuration section name (default: site_info_configuration)
+            cachePath (str, optional):  path to cache directory (default: '.')
+            stashRemotePrefix (str, optional): file name prefix (channel) applied to remote stash file artifacts (default: None)
+        """
+        configPath = kwargs.get("configPath", "exdb-config-example.yml")
+        self.__configName = kwargs.get("configName", "site_info_configuration")
+        mockTopPath = kwargs.get("mockTopPath", None)
+        self.__cfgOb = ConfigUtil(configPath=configPath, defaultSectionName=self.__configName, mockTopPath=mockTopPath)
+        #
+        self.__cachePath = kwargs.get("cachePath", ".")
```

Comment:
Usually, we use a directory called "CACHE", rather than putting all the files in the working directory. Is there a reason you changed that?

Reply:
Hmm, I believe I copied this from another workflow and didn't notice that. Maybe I picked a bad example. I'll take a look.

Reply:
Yeah it would be good to address. I just noticed you also made that change in your last merge to `rcsb.utils.dictionary` (sorry I didn't catch that earlier). Would be good to keep that consistent too.

Reply:
I'll review the other PRs where I made similar changes.

Reply:
Sounds great, thank you!